src/auth/auth.service.ts

```
@@ -15,17 +16,26 @@ import { EmailToken } from './entities/email-token.entity';
 import * as bcrypt from 'bcryptjs';
 import type { RegisterDto } from './dto/register-user.dto';
 import type { CreatePortfolioDto } from './dto/create-portfolio.dto';
-import type { JwtService } from '@nestjs/jwt';
+import { JwtService } from '@nestjs/jwt';
 import { addHours } from 'date-fns';
 import * as crypto from 'crypto';
 import type { PasswordReset } from './entities/password-reset.entity';
 import { PasswordReset } from './entities/password-reset.entity';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'PasswordReset'. Consider to delete it or rename it.

'PasswordReset' is defined here:


(lint/suspicious/noRedeclare)